src/include/units/random.h

```diff
@@ -0,0 +1,453 @@
+#pragma once
+
+#include <random>
+#include <functional>
+
+namespace units {
+
+template<typename Quantity>
+struct uniform_int_distribution : public std::uniform_int_distribution<typename Quantity::rep>
+{
+    using Rep = Quantity::rep;
```

Comment:
We shouldn't use big letters in the code (right now it is used only in concepts and macros).

Reply:
Are you referring to ```Rep``` or ```Quantity```? If ```Rep```, should I just replace that with ```rep```?